views/resume/repo.ejs

```diff
@@ -66,16 +57,49 @@
       </div>
       <main>
          <div class="container">
-            <div class="input-field">
-               <input id="search" type="search" required name="search">
-               <label for="search"><i class="material-icons">search</i></label>
-               <i class="material-icons">close</i>
-            </div>
+  <div class="row">
+    <div class="col s12">
+      <div class="row">
+        <div class="input-field col s12">
+          <i class="material-icons prefix">search</i>
+          <input type="text" id="search" class="search" autocomplete="off" list="languages">
+          <label for="search">Search</label>
+        </div>
+      </div>
+    </div>
+  </div>
+
+
+  <datalist id="languages">
```

Comment:
Some of these aren't necessarily "languages"; I would update the div id to something more accurate like `skills`.